docs/usage.rst

```diff
@@ -47,16 +47,36 @@ We select the :class:`audbackend.backend.FileSystem` backend.
     audbackend.backend.FileSystem.create("./host", "repo")
 
 Once we have an existing repository,
-we can access it by instantiating the backend class.
-In addition,
-we use :class:`audbackend.interface.Unversioned`
-as an interface to the backend.
+we can access it by instantiating the backend class
+and open a connection by calling
+:meth:`audbackend.backend.open`.
+Later on,
+we will use
+:meth:`audbackend.backend.close`
+to close the connection.
+Instead of explicitly calling
+:meth:`audbackend.backend.open`
+and
+:meth:`audbackend.backend.close`
+it is good practise to use a with statement
+(not applicable in a notebook).
```

Comment:
```suggestion
we can access it by instantiating the backend class.
Some backends also need to establish a connection.
This can be achieved using a ``with`` statement,
or by calling ``backend.open()`` at the beginning,
and ``backend.close()`` at the end.
If you are unsure
weither your backend needs to establish a connection,
just always do it.
```